colorbleed/plugins/maya/publish/collect_render_layer_aovs.py

```diff
@@ -41,7 +41,7 @@ def process(self, instance):
 
         self.log.info("Renderer found: {}".format(renderer))
 
-        rp_node_types = {"vray": "VRayRenderElement",
+        rp_node_types = {"vray": ("VRayRenderElement", "VRayRenderElementSet"),
```

Comment:
For code consistency it would be nice to assume all values in this dictionary are lists or tuples. This would also indicate to the next person that it support multiple nodes, which the `type` flag of the `cmds.ls()` supports